### PR TITLE
Change comment body to be text instead of string

### PR DESCRIPTION
**Pull request by emmasax1**
Description and Impact
----------------------
No visible change for production.
Comment bodies are now text instead of strings.

Deploy Plan
-----------
* Deploy with migrations
* Check that database fields are accurate now with two body columns (one string values and one text values)

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

QA Plan
-------
- [x] Test it locally

### CODE REVIEW

File: db/migrate/20160113181700_copy_body_string_to_body_text.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class CopyBodyStringToBodyText < ActiveRecord::Migration
+  def change
+     Hubstats::Comment.update_all("body_text=body")
```
Comment (anfleene) [inline db/migrate/20160113181700_copy_body_string_to_body_text.rb]: the down here is different. It would be "body=body_text"

Comment (emmasax1) [inline db/migrate/20160113181700_copy_body_string_to_body_text.rb]: Should we add that? I figured it might not be important, but I can do it quick.

Comment (anfleene) [inline db/migrate/20160113181700_copy_body_string_to_body_text.rb]: so you should change `change` to be `up` and create a  `down` method that does the reverse.

**Comment by anfleene** [comment]
Looks good :+1:

**Pull request merged**